view/frontend/web/js/view/payment/method-renderer/adyen-pm-method.js

```diff
@@ -37,7 +37,6 @@ define(
         adyenCheckout
     ) {
         'use strict';
-        let popupModal;
```

Comment:
This variable is used in the renderer to hold the name of the DOM element while creating a container for the action component.